### PR TITLE
Improve styles

### PR DESCRIPTION
**Pull request by lucasasecas**
@mravinale, @andresmoschini :
Change boar background.
Add navbar with action buttons.
Change sticky notes styles.

### CODE REVIEW

**Comment by andresmoschini** [comment]
Hi Lucas, I cannot review it during these days, but if you feel enough
confidence or someone else could review it, feel free to merge it.

**Comment by mravinale** [comment]
sure no problem as far I see looks pretty good

**Comment by andresmoschini** [comment]
Hi Mariano,  thanks by the review. Take into account that in our flow, the
merge is done by the same person that creates the PR.

**Comment by mravinale** [comment]
:+1:

File: client/index.html [hunk 1]
```diff
@@ -9,32 +9,37 @@
...
     <div class="board" data-bind="drop: true, foreach: notes">
-      <ul class="note" data-bind="drag: true, style: style">
-        <li><label>Title:<input type="text" data-bind="textInput: title" /></label></li>
-        <li><label>Content:<input type="text" data-bind="textInput: content" /></label></li>
-        <li><label>PosX:<input type="text" data-bind="textInput: posX" /></label></li>
-        <li><label>PosY:<input type="text" data-bind="textInput: posY" /></label></li>
+      <ul class="note" data-bind="drag: true, style: style" >
+        <!-- <li><label>Title:<input type="text" data-bind="textInput: title" /></label></li> -->
+        <li>
+            <h2  data-bind="jeditable: title, jeditableOptions: {}">Title</h2>
```
Comment (andresmoschini) [inline client/index.html:22]: It seems that _ul/li_ has not sense in the new style, what do you think about remove them and simplify note markup?

File: client/index.html [hunk 2]
```diff
@@ -9,32 +9,37 @@
 </head>
 
 <body>
-
+  <ul class="nav-horizontal">
+    <li><a href="#" >Logo</a></li>
```
Comment (andresmoschini) [inline client/index.html:13]: Extra space

```
    <li><a href="#" >Logo</a></li>
                   ^
```

File: client/index.html [hunk 3]
```diff
@@ -9,32 +9,37 @@
...
-        <li><label>Content:<input type="text" data-bind="textInput: content" /></label></li>
-        <li><label>PosX:<input type="text" data-bind="textInput: posX" /></label></li>
-        <li><label>PosY:<input type="text" data-bind="textInput: posY" /></label></li>
+      <ul class="note" data-bind="drag: true, style: style" >
+        <!-- <li><label>Title:<input type="text" data-bind="textInput: title" /></label></li> -->
+        <li>
+            <h2  data-bind="jeditable: title, jeditableOptions: {}">Title</h2>
+            <div class="text-editable" data-bind="jeditable: content, jeditableOptions: {type: 'textarea', submit: 'Ok', cancel: 'Cancel'}" >Inner text</div>
+        </li>
+        <!-- <li><label>PosX:<input type="text" data-bind="textInput: posX" /></label></li>
```
Comment (andresmoschini) [inline client/index.html:25]: What do you think about to remove commented code?

File: client/styles.css [hunk 4]
```diff
@@ -1,17 +1,66 @@
+body{
+  margin: 0;
+  padding: 0;
+    background: url('img/cork.jpg');
```
Comment (andresmoschini) [inline client/styles.css:4]: Indentation.

File: src/view-model.ts [hunk 5]
```diff
@@ -70,6 +73,8 @@ module ViewModel {
       id = id || Utils.randomString();
       var note = new Note();
       note.id = id;
+      note.title( "Title here" );
```
Comment (andresmoschini) [inline src/view-model.ts:76]: It seems that `Title here` is not a real value, could you find another possibility? for example to renter `Title here` when the value is null or empty.

BTW, extra spaces:

```
note.title( "Title here" );
           ^            ^
```

**Comment by andresmoschini** [comment]
Nice work, Lucas.

Do you have a link to test it online?

File: client/index.html (hunk 1 above)
Comment (lmarquezMS) [inline client/index.html:22]: I think that It's depends on whether you just have a unique text or a title and a description.

**Comment by lmarquezMS** [comment]
> Nice work, Lucas.
> 
> Do you have a link to test it online?

Sorry, I haven't time to create a deploy environment

File: client/index.html (hunk 1 above)
Comment (andresmoschini) [inline client/index.html:22]: but is there not only a li element in any case?

Comment (lmarquezMS) [inline client/index.html:22]: You're right. 
I did not remember that title and paragraph are in the same _li_
I think that is not necessary  keep using the list structure.

**Pull request merged**